.github/workflows/release-continuous.yml

```diff
@@ -1,6 +1,8 @@
 name: Publish Any Commit
 on: [push, pull_request]
 
+permissions: {}
+
```

Comment:
I'm not sure about this one, but the others are same with the core repo and is working there.

Reply:
https://github.com/stackblitz-labs/pkg.pr.new/issues/345